addons/gamification/security/ir.model.access.csv

```diff
@@ -28,3 +28,6 @@ badge_user_public,"Badge-user Public",gamification.model_gamification_badge_user
 
 gamification_karma_rank_access_all,gamification.karma.rank.access.all,gamification.model_gamification_karma_rank,,1,0,0,0
 gamification_karma_rank_access_user_manager,gamification.karma.rank.access.user.manager,gamification.model_gamification_karma_rank,base.group_erp_manager,1,1,1,1
+
+gamification_karma_tracking_access_all,gamification.karma.tracking.access.all,gamification.model_gamification_karma_tracking,,1,0,0,0
```

Comment:
I would define this model as purely technical. No access except erp manager. It implies sudo-ing accesses to it and/or providing methods to compute stuff based on it. That way we avoid people trying to read information they should not really read.